config/deprecation-workflow.js

```diff
@@ -2,23 +2,25 @@
 window.deprecationWorkflow = window.deprecationWorkflow || {};
 window.deprecationWorkflow.config = {
   workflow: [
-    { handler: "silence", matchId: "ember-global" },
-    { handler: "silence", matchId: "ember.component.reopen" },
-    { handler: "silence", matchId: "ember-metal.get-with-default" },
+    { handler: "throw", matchId: "ember-global" },
+    { handler: "silence", matchId: "ember.component.reopen" }, // not finish yet
+    { handler: "throw", matchId: "ember-metal.get-with-default" },
     { handler: "silence", matchId: "implicit-injections" },
     {
-      handler: "silence",
+      // Fixed
+      handler: "throw",
       matchId: "ember-views.curly-components.jquery-element",
     },
     {
+      // the issue exists in ember scrollable package
       handler: "silence",
       matchId: "deprecated-run-loop-and-computed-dot-access",
     },
-    { handler: "silence", matchId: "ember-utils.try-invoke" },
+    // { handler: "throw", matchId: "ember-utils.try-invoke" }, something in ember core
```

Comment:
Please add comment (*This error is caused by `ember-parachute` and will persist until we have refactored all controllers and routes to **not** use `ember-parachute`*.  Something like that.)  but leave it as `silenced`.